.github/ISSUE_TEMPLATE/project.md

```diff
@@ -21,35 +21,56 @@ assignees: ''
 
 <!-- Please fill out the following questions to rationalize the prioritization of this project -->
 
-- Does this serve our mission?
+- Does this serve our mission
 
 - Will this help to make us financially sustainable within a year?
```

Comment:
as I understnad it this was a stated goal (that maybe we have reached?) but I would love to link to a set of longer-lived goals that we're working toward that helps us acheive our mission.

Reply:
I like that - maybe we can also link to the values section in docs, or bring some of that working into the template.